src/components/app/app.html

```diff
@@ -2,6 +2,11 @@
   <div class="app__drawer-point"></div>
   <div class="app__container">
     <div class="app__container-toolbar-point"></div>
-    <div class="app__container-content"></div>
+    <div class="app__container-content">
+      <button class="mdc-fab material-icons app__add-transaction-dialog-activation" aria-label="Favorite">
+          <span class="mdc-fab__icon">add</span>
+        </button>
+      <div class="app__add-transaction-dialog"></div>
+    </div>
```

Comment:
maybe it is better to make `<div class="app__container-content">` as `<main class="app__container-content">`
and replace  button and `<div class="app__add-transaction-dialog">` outside of this container in `<div class="app__container">`?